<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsTextMessageStyle</class>
 <widget class="QFrame" name="CSettingsTextMessageStyle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text message style</string>
  </property>
  <layout class="QHBoxLayout" name="hl_SettingsTextMessageStyle">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QPushButton" name="pb_FontPlus">
     <property name="toolTip">
      <string>font enlarge (plus)</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/pastel/icons/pastel/16/font-enlarge.png</normaloff>:/pastel/icons/pastel/16/font-enlarge.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_FontMinus">
     <property name="toolTip">
      <string>font shrink (minus)</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/pastel/icons/pastel/16/font-shrink.png</normaloff>:/pastel/icons/pastel/16/font-shrink.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_Reset">
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_Style">
     <property name="text">
      <string>style</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_Font">
     <property name="text">
      <string>font</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_FontPlus</tabstop>
  <tabstop>pb_FontMinus</tabstop>
  <tabstop>pb_Reset</tabstop>
  <tabstop>pb_Style</tabstop>
  <tabstop>pb_Font</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
